examples/components/pageviewer.js

```diff
@@ -47,28 +47,29 @@ const loadingTask = pdfjsLib.getDocument({
   cMapPacked: CMAP_PACKED,
   enableXfa: ENABLE_XFA,
 });
-loadingTask.promise.then(function (pdfDocument) {
+
```

Comment:
Given that the existing code doesn't have a blank-line here, let's not add one either.

Reply:
Updated